roles/import-from-url/tasks/provision.yml

```diff
@@ -35,6 +35,31 @@
 - name: Provision PVC
   command: "oc apply -f /tmp/pvc.yml"
 
+- name: set pvc status jsonpath selectors
+  set_fact:
+    import_status_path: .metadata.annotations.kubevirt\.io/storage\.import\.pod\.phase
+    pvc_status_path: .status.phase
+
+- Name: Check PVC bound status
+  command: "oc get pvc {{ vm_name }}-disk-01 -o jsonpath='{ {{pvc_status_path}} }'"
+  register: pvc_bound_status
+  until: pvc_bound_status.stdout == 'Bound'
+  retries: 10
```

Comment:
What happens if the pvc is not bound within given time?

Reply:
Then the ansible command will fail, and the whole bundle will fail (as the PVC can't get bound, and thus the import can't happen). 10 seconds is very generous for a PVC bind.

Reply:
Actually I've seen it take longer than that for cinder storage.  Let's make this at least 60 seconds but better would be 3 minutes.